src/Symfony/Component/PropertyAccess/Annotation/PropertyAccessor.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyAccess\Annotation;
+
+/**
+ * Property accessor configuration annotation.
+ *
+ * @Annotation
+ * @Target({"PROPERTY"})
+ *
+ * @author Luis Ramón López <lrlopez@gmail.com>
+ */
+class PropertyAccessor extends ConfigurationAnnotation
```

Comment:
I would name this to "Property". When using an annotation, annotation names should typically read as "this is <name>" or "this is a <name>":

~~~php
/**
 * @PropertyAccessor
 */
private $field.
~~~

The sentence "$field is a property accessor" is technically wrong. The sentence "$field is a property" (that is accessible using PropertyAccess) is correct however.